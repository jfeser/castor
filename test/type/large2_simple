    aorderedidx(atuple([ascalar(date("1998-01-01") as o_orderdate),
                        ascalar(date("2005-01-01") as o_orderdate)], concat),
                      atuple([alist(select([l_extendedprice, l_discount],
                                      join(((o_orderdate = 0.o_orderdate) &&
                                           (((l_returnflag = "R")) &&
                                           (l_orderkey = o_orderkey))),
                                        lineitem,
                                        orders)),
                                atuple([ascalar(0.l_extendedprice),
                                        ascalar(0.l_discount)],
                                  cross))],
                        cross),
      >= param0, < (param0 + month(3)))
