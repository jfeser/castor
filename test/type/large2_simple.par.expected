(OrderedIdxT
 ((DateT ((range (Interval 10227 12784))))
  (TupleT
   (((ListT
      ((TupleT
        (((FixedT ((value ((range Top) (scale 1)))))
          (FixedT ((value ((range Top) (scale 1))))))
         ((kind Cross))))
       ((count (Interval 0 0))))))
    ((kind Cross))))
  ((key_count (Interval 2 2)))))
